app/workers/daily_grouping_updater.rb

```diff
@@ -0,0 +1,27 @@
+class DailyGroupingUpdater
+  include Sidekiq::Worker
+
+  def initialize
+    @system_account = Watcher.find_by_email("kairos-robot@omadahealth.com")
```

Comment:
I would really prefer keeping kairos/omada specific things out of this repo.  I also don't see anywhere that this account is created.